### PR TITLE
add azure workload Id destructive ci.

### PR DESCRIPTION
**Pull request by huangmingxia**
[CCO-380](https://issues.redhat.com/browse/CCO-380)
[CCO-403](https://issues.redhat.com/browse/CCO-403)
 Add the destructive ci .

### CODE REVIEW

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-arm64-nightly-azure-ipi-workload-identity-tp-p1-f14-destructive

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-azure-ipi-workload-identity-tp-p1-f14-destructive

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-arm64-nightly-azure-ipi-workload-identity-tp-p1-f28-destructive

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-azure-ipi-workload-identity-tp-p1-f28-destructive

**Comment by huangmingxia** [comment]
Hi @jhou1, would you mind help review this pr when you free?  Thanks for your time.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 1]
```diff
@@ -1869,6 +1869,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity
+- as: azure-ipi-workload-identity-tp-p1-f28-destructive
+  cron: 37 2 5 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @azure-ipi and @network-ovnkubernetes and not @fips
```
Comment (LiangquanLi930) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1878]: `@network-ovnkubernetes` This workflow network type is ovn?

Comment (huangmingxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1878]: @LiangquanLi930 Yes. Is ovn.

**Comment by huangmingxia** [comment]
@jianlinliu @yunjiang29 @jhou1  could you help review this pr when you free? Thanks.

**Comment by huangmingxia** [comment]
Hi@Xia-Zhao-rh, could you help review this pr? thanks.

**Comment by jianlinliu** [comment]
cc @jhou1 to review and approve.

**Comment by jianlinliu** [comment]
@huangmingxia can you squash the multiple commits into one?

**Comment by huangmingxia** [comment]
@jianlinliu No problem~ commits are squashed.

**Pull request merged**